data_steward/utils/bq.py

```diff
@@ -5,28 +5,37 @@
 import logging
 import os
 
-from google.api_core.exceptions import GoogleAPIError, BadRequest
 # Third-party imports
+from google.api_core.exceptions import GoogleAPIError, BadRequest
 from google.cloud import bigquery
+import google.auth
```

Comment:
nit: unused import

Reply:
The `google.auth.default()` in `get_client()` is using this. I'll make this `from google.auth import default`

Reply:
Sorry, my mistake- looks like a false alarm from the IDE